Altis_Life.Altis/config/Config_Master.hpp

```diff
@@ -157,6 +157,8 @@ class Life_Settings {
     vehicle_storage_fee_multiplier = .2; //Pull from garage cost --> Cost takes the playersides Buy Price * multiplier
     vehicle_cop_impound_multiplier = .1; //TO AVOID EXPLOITS NEVER SET HIGHER THAN A PURCHASE/RENTAL multipler!   Payout = Config_vehicle Price * multiplier
 
+    command_view = true; // -- Set to false in order to enable command view
```

Comment:
Could you better explain what the command view is in the comment? A lot of people may not know what it is.